HeterogeneousCore/CUDAUtilities/interface/GPUSimpleVector.h

```diff
@@ -83,6 +83,11 @@ template <class T> struct SimpleVector {
   __inline__ __host__ __device__ int capacity() const { return m_capacity; }
 
   __inline__ __host__ __device__ T *data() const { return m_data; }
+    
+  __inline__ __host__ __device__ void set_size(int size) { m_size = size; }
```

Comment:
I would call this `resize()`, unless people find it confusing ?

Reply:
I agree on `resize()`, the effect would anyway be similar to `std::vector`. (or remove as it is not currently used).